lms/urls.py

```diff
@@ -427,6 +427,8 @@
         # Student Notes
         url(r'^courses/{}/edxnotes'.format(settings.COURSE_ID_PATTERN),
             include('edxnotes.urls'), name="edxnotes_endpoints"),
+
+        url(r'^api/v1/branding/', include('branding.api_urls')),
```

Comment:
URL namespaces allow you to uniquely reverse named URL patterns even if different applications use the same URL names (e.g., `data`); reversing a namespaced URL is also very readable. If you were to assign the branding API to the namespace `branding`, you could `reverse('branding:footer:data')` or `reverse('branding:header:data')`, helping to keep the API's URL structure clean as it grows.